packages/inventory/src/services/reservation-item.ts

```diff
@@ -321,4 +355,35 @@ export default class ReservationItemService {
       id: reservationItemId,
     })
   }
+
+  private prepareSearchQuery(
```

Comment:
todo: I think that it would make sense to move it to the buildQuery utils file and use it from there something that just return the appropriate FindOperator that we can then use elsewhere. I can see that used in other places instead of building it in place and just pass the result of that util. wdyt?